functional/security/Crypto/playlist.xml

```diff
@@ -49,5 +49,47 @@
 		<groups>
 			<group>functional</group>
 		</groups>
+		<vendors>
+			<vendor>redhat</vendor>
```

Comment:
Can this be make opposite? To enable this main for all, except the few broken, and enable  CryptoTests_jtreg only for  the zOs or whereevere the main rnner do not fit?

Reply:
> Can this be make opposite? To enable this main for all, except the few broken, and enable CryptoTests_jtreg only for the zOs or whereevere the main rnner do not fit?

If this PR would have not moved the default exclusion in favour of itself it would be awesome temporary workaround for affected platforms before the real issue s fixed.